tests/container_checker/test_container_checker.py

```diff
@@ -229,10 +229,12 @@ def test_container_checker(duthosts, enum_dut_feature_container, rand_selected_d
 
     pytest_require(container_name not in skip_containers,
                    "Container '{}' is skipped for testing.".format(container_name))
-    stop_container(duthost, container_name)
 
     loganalyzer.expect_regex = get_expected_alerting_message(container_name)
-    with loganalyzer:
-        # Wait for 1 minutes such that Monit has a chance to write alerting message into syslog.
-        logger.info("Sleep 1 minutes to wait for the alerting message...")
-        time.sleep(70)
+
+    try:
+        loganalyzer.disable_log_rotate()
+        with loganalyzer:
+            stop_container(duthost, container_name)
```

Comment:
i think we need to explicitly wait for 70 seconds here, since that will ensure the monit message in syslog

Reply:
The wait was done through wait expected events.